modules/generative-openai/module.go

```diff
@@ -67,8 +67,9 @@ func (m *GenerativeOpenAIModule) initAdditional(ctx context.Context,
 ) error {
 	openAIApiKey := os.Getenv("OPENAI_APIKEY")
 	azureApiKey := os.Getenv("AZURE_APIKEY")
+	openAIBaseUrl := os.Getenv("OPENAI_BASE_URL")
 
-	client := clients.New(openAIApiKey, azureApiKey, logger)
+	client := clients.New(openAIApiKey, azureApiKey, openAIBaseUrl, logger)
```

Comment:
we decided to pass the `baseURL` using `class settings` so you should remove the possibility of setting it through env settings

Reply:
We should remove the `baseURL` from client, it should work by default with `OpenAI` baseURL only with moduleConfig explicit configuration one should be able to change `baseURL`